backend/config/settings.py

```diff
@@ -30,7 +30,7 @@
 DEBUG = True
 
 ALLOWED_HOSTS = ['127.0.0.1', 'forum-prod-api.herokuapp.com']
-CORS_ALLOW_ALL_ORIGINS = False
```

Comment:
Do we need this change?

Reply:
We need to allow cross-origins if we don't it gives errors while deployment.